src/app/components/charts/chart-volume/chart-volume.component.ts

```diff
@@ -0,0 +1,70 @@
+import { ChangeDetectionStrategy, Component } from '@angular/core'
+import * as d3 from 'd3'
+
+import { ChartBaseComponent } from '../chart-base/chart-base.component'
+
+@Component({
+  selector: 'app-chart-volume',
+  templateUrl: '../charts.common.html',
+  styleUrls: ['./chart-volume.component.scss'],
+  changeDetection: ChangeDetectionStrategy.OnPush
+})
+export class ChartVolumeComponent extends ChartBaseComponent {
+  dates: Date[]
+
+  svg: any
```

Comment:
remove values that are part of the superclass and are not being changed in this class.